rpcs3/rpcs3qt/tooltips.h

```diff
@@ -181,6 +181,7 @@ class Tooltips : public QObject
 		const QString camera            = tr("Camera support is not implemented, leave this on null.");
 		const QString camera_type       = tr("Camera support is not implemented, leave this on unknown.");
 		const QString move              = tr("PlayStation Move support.\nFake: Experimental! This maps Move controls to DS3 controller mappings.\nMouse: Emulate PSMove with Mouse handler.");
+		const QString buzz              = tr("Emulate Buzz! controllers.");
```

Comment:
Here is where you actually need to be descriptive about the setting. This is the GUI tooltip.